### ISSUE TITLE
Domain Adaptation using SeeingThroughFog

### ISSUE THREAD

**Issue by Rolandxx7** [princeton-computational-imaging/SeeingThroughFog#57]
Dear Mario Bijelic, thanks for your great job! Recently, I am trying to use CyCADA to realize domain adaptation from clear winter captures to adverse weather scenes, as shown on your official website. However,  my CyCADA generates snow with poor results after training on the SeeingThroughFog for 70 epochs. So I would like to ask you for help. Could you tell me about your training steps and training parameter settings？

<img width="460" alt="1662179604106" src="https://user-images.githubusercontent.com/76581571/188255953-9cfd7f4a-2eee-4e52-8cd7-890dfb7252ca.png">

**Comment by MarioBijelic** [comment]
Dear @Rolandxx7,

as domain adaptation did not need labels for supervision I was able to use way more unlabelled data then labeled data for the supervised methods, which are at the moment not available online due to space constraints. I have experienced that with less data you have to constrain the problems more. Hence throwing out the foggy data and constraining methods only to tasks clear to foggy or clear to snowy conditions helps. Don't hesitate to reach out via mail directly.

Best, 
Mario
